libs/dbus_util/tests/examples/example_sml_interface.cpp

```diff
@@ -1,25 +1,61 @@
+#include <memory>
+#include <string>
+#include <string_view>
+
 #include <boost/asio.hpp>
 #include <boost/sml.hpp>
-#include <memory>
-#include <sdbusplus/asio/connection.hpp>
 #include <sdbusplus/asio/object_server.hpp>
+
+#include <tfc/ipc.hpp>
+#include <tfc/progbase.hpp>
+
 #include <tfc/dbus/sml_interface.hpp>
-struct state_machine {
+#include <tfc/dbus/string_maker.hpp>
+
+struct running {
```

Comment:
I would say it is conflicting to call state running and event running, event can be named run and event stop

Reply:
Changing event names to `run` and `stop`